auth/build.gradle.kts

```diff
@@ -104,6 +146,66 @@ tasks {
     val iosX64Test by existing(KotlinNativeSimulatorTest::class) {
         filter.excludeTestsMatching("com.liftric.auth.AuthHandlerIntegrationTests")
     }
+
+    val testSecrets by creating(GetVaultSecretTask::class) {
+        secretPath.set("secret/apps/smartest/shared/cognito")
+    }
+
+    val createJsEnvHack by creating {
```

Comment:
We could use https://github.com/yshrsmz/BuildKonfig for this. But I'm fine with rolling this on our own.

Reply:
For production code sure! But for getting two test variables into the test targets, I think this is good enough :)